packages/flutter_plugin_android_lifecycle/pubspec.yaml

```diff
@@ -22,3 +22,7 @@ dependencies:
 dev_dependencies:
   flutter_test:
     sdk: flutter
+
+topics:
+  - android
+  - plugins
```

Comment:
This is a topic because this plugin is for the purpose of writing other plugins against, not because it is a plugin. (Same with `plugin_platform_interface` having the `plugins` topic.)

Reply:
I also thought that `plugins` meant "this is a Flutter plugin". I doubt we'll be able to maintain this distinction, plugin authors might make the same mistake and use this topic incorrectly. Some potential alternatives that might be clearer:

* `plugin-development`
* `plugin-library`
* `plugin-helper`
* `plugin-dependency`

Reply:
I like `plugin-development`; I'll update both accordingly.

Reply:
Seems like pigeon should have that also

Reply:
Added. I think I didn't have it originally because Pigeon isn't just for plugins, but it's a common use case certainly so we may as well include it.